23-freight-car-door-motion-and-location/config.json

```diff
@@ -1,5 +1,5 @@
 # set your own product UID and serial number
-{ "req": "hub.set", "sn": "nf23", "product": "com.blues.mat:test" }
+{ "req": "hub.set", "sn": "my-serial-number", "product": "my-product-uid" }
```

Comment:
Nitpick, but the placeholders referenced in the README.md are a-serial-number and com.your-company:your-product-name.

Reply:
fixed